<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRangParaSetWidget</class>
 <widget class="BaseDlg" name="CRangParaSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRangParaSetWidget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>95</y>
     <width>536</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_11" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>50</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>8</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>是否缺省</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="auto_default_cbox">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <item>
              <property name="text">
               <string>Y</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>性别</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gender_cbox">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>危急低值</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ref_critical_lower_rang">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="placeholderText">
              <string>低值</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>8</number>
           </property>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>样本类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="samp_source_cbox">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>年龄</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>7</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lower_age_edit">
               <property name="minimumSize">
                <size>
                 <width>55</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>55</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maxLength">
                <number>6</number>
               </property>
               <property name="placeholderText">
                <string>下限</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>—</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="dividers" stdset="0">
                <string>gray</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="upper_age_edit">
               <property name="minimumSize">
                <size>
                 <width>55</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>55</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maxLength">
                <number>6</number>
               </property>
               <property name="placeholderText">
                <string>上限</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="age_unit_cbox">
               <property name="minimumSize">
                <size>
                 <width>98</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>88</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>危急高值</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ref_critical_upper_rang">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="placeholderText">
              <string>高值</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>参考值范围</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="ref_lower_age_edit">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="placeholderText">
              <string>下限</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>—</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="dividers" stdset="0">
              <string>gray</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ref_upper_age_edit">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>40</height>
              </size>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="placeholderText">
              <string>上限</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="str_refrange_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>14</number>
      </property>
      <property name="topMargin">
       <number>17</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ok_btn">
        <property name="minimumSize">
         <size>
          <width>116</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_btn">
        <property name="minimumSize">
         <size>
          <width>116</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="buttons" stdset="0">
         <string>btn_dlg</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>auto_default_cbox</tabstop>
  <tabstop>samp_source_cbox</tabstop>
  <tabstop>gender_cbox</tabstop>
  <tabstop>lower_age_edit</tabstop>
  <tabstop>upper_age_edit</tabstop>
  <tabstop>age_unit_cbox</tabstop>
  <tabstop>ref_critical_lower_rang</tabstop>
  <tabstop>ref_critical_upper_rang</tabstop>
  <tabstop>ref_lower_age_edit</tabstop>
  <tabstop>ref_upper_age_edit</tabstop>
  <tabstop>ok_btn</tabstop>
  <tabstop>close_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>CRangParaSetWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
